DuckDuckGoTests/VariantTests.swift

```diff
@@ -22,8 +22,10 @@ import XCTest
 
 class VariantTests: XCTestCase {
 
-    func testDefaultVariantsPercentageTotalIs100() {
-        XCTAssertEqual(100, Variant.defaultVariants.reduce(0, { $0 + $1.percent }))
+    /// When there's an experiment, it should add up to 100%, otherwise zero
```

Comment:
Unrelated to this review but worth noting while I'm here. We don't have the requirement for experiments to add up to 100 in android it would be good to not need that.

Reply:
💯 agreed - will change that in upcoming task. It's a bit annoying having to change this much code when adding/removing variants.